src/main/java/com/commercetools/sync/inventories/InventorySync.java

```diff
@@ -25,34 +25,36 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import java.util.Set;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
 import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
+import static com.commercetools.sync.inventories.InventorySyncMessages.CTP_INVENTORY_ENTRY_CREATE_FAILED;
```

Comment:
why extract the messages to a separate class, when they are only used by this class? If it is because too many messages, then we need a clear strategy about when do we decide when to separate messages into separate classes and when not? A lot of other classes in the project have messages in them but are not separated. Shall we also separate when it's one message?